templates/home-page.html

```diff
@@ -1,7 +1,7 @@
 {% extends 'base.html' %}
 
 {% block head %}
-	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/home-page.css') }}">
+	<link defer rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/home-page.css') }}">
```

Comment:
This line was never to be changed, @Haroonchik. Please let the original line be.

The `script` tag needs to be included below this line with an added `defer` attribute to it.

Reply:
Ohhh, sry. I'll fix it now.